content/docs/rosa/alb-nlb-ossm-e2e/index.md

```diff
@@ -0,0 +1,530 @@
+  
+```bash
+oc get sub elasticsearch-operator -n openshift-operators --output jsonpath='{.status.conditions[?(@.type == "CatalogSourcesUnhealthy")].message}'
+```
+
+```bash
+oc get sub jaeger-product  -n openshift-operators --output jsonpath='{.status.conditions[?(@.type == "CatalogSourcesUnhealthy")].message}'
+```
+
+```bash
+oc get sub kiali-ossm  -n openshift-operators --output jsonpath='{.status.conditions[?(@.type == "CatalogSourcesUnhealthy")].message}'
+```
+
+```bash
+oc get sub servicemeshoperator -n openshift-operators --output jsonpath='{.status.conditions[?(@.type == "CatalogSourcesUnhealthy")].message}'
+```
+
+### Create a service mesh instance
+   
+Create a service mesh instance. We assign a static IP addresses(10.201.1.199, 10.201.0.199,10.201.2.199) to the istio-ingress NLB
+
+```bash
+cat << EOF | oc apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ossm
+--- 
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  namespace: ossm
+  name: ossm-e2e-encryption
+spec:
+  cluster:
+    name: ossm-e2e-encryption
+  addons:
+    grafana:
+      enabled: true
+    jaeger:
+      install:
+        storage:
+          type: Memory
+    kiali:
+      enabled: true
+    prometheus:
+      enabled: true
+  policy:
+    type: Istiod
+  telemetry:
+    type: Istiod
+  tracing:
+    sampling: 10000
+    type: Jaeger
+  runtime:
+    defaults:
+      container:
+        imagePullPolicy: Always
+  proxy:
+    accessLogging:
+      file:
+        name: /dev/stdout        
+  security:
+    identity:
+      type: ThirdParty
+      thirdParty:
+        audience: istio-ca
+  gateways:
+    ingress: # istio-ingressgateway
+      service:
+        metadata:
+            annotations:
+                  service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: instance
+                  service.beta.kubernetes.io/aws-load-balancer-scheme: internal
+                  service.beta.kubernetes.io/aws-load-balancer-private-ipv4-addresses: 10.201.0.199, 10.201.1.199, 10.201.2.199
+                  service.beta.kubernetes.io/aws-load-balancer-type: external
+        type: LoadBalancer
+EOF
+```
+
+**Note**: By default, NodePorts in a ROSA are not accessible outside of the cluster. To enable access to NodePorts from a load balancer, you need to modify the security group associated with your cluster and allow the port range 30000-32767 to receive traffic.
+
+### Deploy AWS Load Balancer Operator (ALBO)
+
+Use this [mobb.ninja](https://mobb.ninja/docs/rosa/aws-load-balancer-operator/) to install ALB operator on ROSA cluster or use the following snippet or run the  [script](./alb-operator/deploy-aws-lbo.sh)
```

Comment:
Use the following snippet or run the script in the cloned repo ...   see [mobb.ninja] article for more details about the ALB operator and its installtion.

Reply:
done